src/shared/context/theme-provider/ThemeProvider.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import themeStyles from '../../styles';
+import PropTypes from 'prop-types';
+
+export const ThemeContext = React.createContext();
+
+const ThemeContextProvider = ({ children }) => (
+    <ThemeContext.Provider value={ { themeStyles } }>
```

Comment:
I don't remember why we decided to pass an object instead of passing the styles directly. 🤔 
Wouldn't the following be enough:

```suggestion
    <ThemeContext.Provider value={ themeStyles }>
```

?

Reply:
To be prepared for future improvements where you will need to pass a setter for example to obtain themeStyles for a certain theme.

Reply:
Yep, you're right! Thanks!